dbt/contracts/graph/parsed.py

```diff
@@ -481,6 +569,19 @@ def _find_by_name(self, name, package, subgraph, nodetype):
             package,
             nodetype)
 
+    def find_docs_by_name(self, name, package=None):
+        for unique_id, doc in self.docs.items():
+            parts = unique_id.split('.')
+            if len(parts) != 2:
```

Comment:
am i crazy or should `parts` have 3 items here? Something like `docs.project_name.docs_name`?